pkg/version/version.go

```diff
@@ -19,9 +19,10 @@ package version
 import "fmt"
 
 // The below variables will be overrriden using ldflags set by goreleaser during the build process
+
 var VERSION = "DEV"
-var GIT_COMMIT = "NONE"
-var BUILD_DATE = "UNKNOWN"
+var GIT_COMMIT = "NONE"    //nolint:stylecheck // Validate goreleaser works find after the change
+var BUILD_DATE = "UNKNOWN" //nolint:stylecheck // Validate goreleaser works find after the change
```

Comment:
Why do we need to ignore the checks? We can add the reason in the comment?

Reply:
`// Validate goreleaser works find after the change` Is it a TODO comment? Something we are supposed to do as a part of this PR? I am assuming it will be removed once we validate that, right>

Reply:
`GIT_COMMIT` and `BUILD_DATE` is referred only in VersionString functions [here](https://github.com/kanisterio/kanister/pull/2505/files#diff-73b8ec8a05213857da6e9598e34c5d7a18a6e5349250908ae6710578236977f2L28) and its value being used. Do we really need to test if goreleaser works fine? Beacuse value as `NONE` and `UNKNOWN` will be used in reference to the var names.

Reply:
These fields do seen to be used from goreleaser https://github.com/kanisterio/kanister/blob/1ebea1abd67ab4c8070a4385d9609f125438c33e/.goreleaser.yml#L11-L13

Reply:
ACK.Checking the testing method.